ccd-definition/RoleToAccessProfiles/RoleToAccessProfiles-prod.json

```diff
@@ -1,20 +1,3 @@
 [
```

Comment:
Can you move these change to ccd-definition/RoleToAccessProfiles/RoleToAccessProfiles.json and delete this file please